pydrive2/files.py

```diff
@@ -34,6 +35,10 @@ def __init__(self, http_error):
         # Initialize args for backward compatibility
         super().__init__(http_error)
 
+    def GetField(self, field):
```

Comment:
we can simplify DVC code after release probably

Reply:
yes, on my post-merge list :)